addons/account/models/account_move.py

```diff
@@ -4482,6 +4482,21 @@ def _generate_pdf_and_send_invoice(self, template, force_synchronous=True, allow
         composer = self.env['account.move.send'].create(composer_vals)
         return composer.action_send_and_print(force_synchronous=force_synchronous, allow_fallback_pdf=allow_fallback_pdf, bypass_download=bypass_download)
 
+    def _get_portal_download(self):
```

Comment:
```suggestion
    def _get_invoice_legal_documents(self):
```
Let's name it with something less specific.

Reply:
Done :wink: